docs/global-styles/nprogress.js

```diff
@@ -1,7 +1,5 @@
 import * as Constants from '~/common/constants';
 
 export const globalNProgress = `
```

Comment:
Assuming this file is vendored and we have no intent on editing it directly, add it to .prettierignore and revert the formatting changes.